docs/io.md

```diff
@@ -15,17 +15,17 @@ data = mmcv.load('test.yaml')
 data = mmcv.load('test.pkl')
 # load data from a file-like object
 with open('test.json', 'r') as f:
-    data = mmcv.load(f)
+    data = mmcv.load(f, file_format='json')
```

Comment:
This was intended. Just to show the common usage without explicit `file_format` arguments.